resources/sql/queries.sql

```diff
@@ -54,6 +54,39 @@ WHERE 1=1
 /*~ ) ~*/
 ORDER BY ci.id;
 
+-- :name get-all-catalogue-items :? :*
+SELECT 
+  id, 
+  resid AS "resource-id", 
+  wfid AS "workflow-id", 
+  formid AS "form-id", 
+  start, 
+  endt AS "end", 
+  enabled, 
+  archived, 
+  organization AS "organization-id",
+  catalogueitemdata::TEXT
+FROM catalogue_item;
```

Comment:
we could remove the old `get-catalogue-items` function now that it is no longer used?